sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Services/Protocols/ClientCertificateInfo.cs

```diff
@@ -2,12 +2,18 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
 
 namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
```

Comment:
>Microsoft.Azure.WebJobs.Extensions.WebPubSub [](http://example.com/codeflow?start=10&length=44)

how about using a more general namespace, e.g `Azure.Messaging.WebPubSub` so that later the data model can be shared in middleware?

Reply:
Make sense. Does the service request(`ConnectEventRequest`, `DisconnectEventRequest`) need in middleware as well?

Reply:
yeah I think all the data models can be reused

Reply:
Leave `MessageEventRequest` in the extensions cause it's not directly deserialize from request body but combined with content-type. Let me know if you'd think this can be commonly used as well.

Reply:
As offline synced, move all of request related types to common namespace.